admin/views/redirect-delay.ejs

```diff
@@ -0,0 +1,8 @@
+<meta http-equiv="refresh" content="<%= WaitTimeBeforeRedirectInSeconds %>;url=<%= url %>" />
+<main class="govuk-main-wrapper" id="main-content" role="main">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds wait-redirect">
+            <h1 class="govuk-heading-xl">Please wait, redirecting...</h1>
```

Comment:
Not sure if you were present, but there was some discussion around the language to use here.  You should seek clarification but the view was that something along the lines of ‘saving changes’ would be best